src/team_lib/hier/xccl_hier_schedule_allreduce.c

```diff
@@ -11,6 +11,7 @@
 #include <stdlib.h>
 #include "xccl_hier_schedule.h"
 #include "xccl_hier_team.h"
+#include "xccl_hier_task_schedule.h"
 
 xccl_status_t build_allreduce_schedule(xccl_hier_team_t *team, xccl_coll_op_args_t coll,
```

Comment:
should we remove old build schedule functions, since we don't use them anymore?
@vspetrov wdyt?

Reply:
@vspetrov ?

Reply:
we can remove them. Just need to keep in mind that new set of schedule routines does not support fragmentation for broadcast. Need to clearly understand how we will implement it with the new framework.

Reply:
removed from schedule_allreduce